### PR TITLE
Master

### PR DESCRIPTION
**Pull request by samsarge**
I can't be waiting any longer to submit a pull request, have to get my place! Had a couple of troubles along the way but added a few other nice features to hopefully make up for it! 

> Finished all tasks
> Used JQuery to program some changes / features which I wanted to do in Ruby but couldn't figure out
> I will be doing the TeamTreehouse course on Rails to see if I can get round to adding these features in Ruby and not JQuery next time.

### CODE REVIEW

File: primes.rb [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+
+primes = []
+for i in 2...1000
+   primes.push(i) if self.sum_to(i)
+end
+#Prints the sum of the range - NOTE TO SELF - look into "inject" method more.
+primes.inject(:+)
+ end
+
+end
```
Comment (dmgarland) [inline primes.rb]: Looks like you're embedding a method within a method here... check your indentation and you'll spot code mistakes more easily.

File: primes.rb [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
-  end
-end
\ No newline at end of file
+  def self.sum_to(limit)
+  # TODO - add your prime number solution here...
+    require 'prime'
```
Comment (dmgarland) [inline primes.rb]: Maybe have a go without the core libraries... show us what you can do with Ruby!

File: primes.rb (hunk 1 above)
Comment (samsarge) [inline primes.rb]: Will jump onto it first thing after work tomorrow! Thanks.

File: primes.rb (hunk 2 above)
Comment (samsarge) [inline primes.rb]: Cheers, will fix it ASAP.

File: primes.rb (hunk 1 above)
Comment (samsarge) [inline primes.rb]: Cheers! Will fix it ASAP.

Comment (samsarge) [inline primes.rb]: I fixed this and completely rewrote the code, looks better, works better - no bug on refresh. Not sure how I got it so wrong in the first place! Pushing changes now but it's taking a while :(

**Pull request closed without merging**